tests/index.ts

```diff
@@ -12,17 +12,29 @@ import {
   TEST_DATA,
 } from "tests/utils";
 
+import { FileIssue } from "./types";
+
 // trunk-ignore(eslint/@typescript-eslint/no-unused-vars): Define the matcher as extracted from dependency
 const toMatchSpecificSnapshot = specific_snapshot.toMatchSpecificSnapshot;
 
 // The underlying implementation of jest-specific-snapshot supports forwarding additional arguments from
 // `toMatchSpecificSnapshot` and appending them to `toMatchSnapshot`, but its type declarations do not explicitly
 // support this. Adding this patch/override allows us to call it with custom matchers, as in the case of `landingStateWrapper`.
+type OwnMatcher<Params extends unknown[]> = (
+  this: jest.MatcherContext,
+  actual: unknown,
+  ...params: Params
+) => jest.CustomMatcherResult;
+
 declare global {
-  // trunk-ignore(eslint/@typescript-eslint/no-namespace)
+  // eslint-disable-next-line @typescript-eslint/no-namespace
```

Comment:
I prefer doing the eslint ignore (rather than trunk) directly here since the extension's warning is quite glaring.